java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java

```diff
@@ -906,137 +992,6 @@ public void testRemoveSpecificAPIpatternsJUnit() throws IOException {
         .doTest();
   }
 
-  @Test
-  public void testRemoveSpecificAPIpatterns() throws IOException {
```

Comment:
In my previous [PR](https://github.com/uber/piranha/pull/156/commits/1bd399497881e45ad9329c99f856cff948bef02e), I had split this test case into smaller test case. But I had not deleted this test case.